src/lamp_py/ingestion_tm/jobs/parition_table.py

```diff
@@ -12,61 +12,33 @@
 from lamp_py.ingestion_tm.tm_export import TMExport
 from lamp_py.mssql.mssql_utils import MSSQLManager
 from lamp_py.runtime_utils.process_logger import ProcessLogger
+from lamp_py.runtime_utils.remote_files import RemoteFileLocations, S3Location
+
 from lamp_py.aws.s3 import (
     file_list_from_s3,
     upload_file,
     object_metadata,
 )
 
 
-class TMDailyLogStopCrossing(TMExport):
-    """Export STOP_CROSSING table from TMDailyLog"""
-
-    def __init__(self) -> None:
-        TMExport.__init__(self)
+class TMDailyTable(TMExport):
+    """Export Daily table from TMDailyLog"""
```

Comment:
i converted this to a base class that other daily log tables can use for export.